File: 1-Iniciante/Ex1007.java
import java.util.Scanner;

public class Ex1007 { //para ser aceito no URI mude o nome da classe para Main
    public static void main(String args[]) {
        /* variaveis */
        Scanner scanner = new Scanner(System.in);
        int a, b, c, d, resultado;

        /* entrada */
        a = scanner.nextInt();
        b = scanner.nextInt();
        c = scanner.nextInt();
        d = scanner.nextInt();

        /* processamento */
        resultado = a * b - c * d;

        /* saida */
        System.out.println("DIFERENCA = " + resultado);
        scanner.close();
    }
}

File: 1-Iniciante/Ex1007.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val a = scanner.nextInt()
    val b = scanner.nextInt()
    val c = scanner.nextInt()
    val d = scanner.nextInt()

    val resultado = a * b - c * d

    println("DIFERENCA = $resultado")
}
